apps/mobile/src/components/developer-console/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { useTheme } from '@shopify/restyle';
+
+import { Box, Theme } from '@leather.io/ui/native';
+
+import { HEADER_HEIGHT } from '../blurred-header';
+import { ApproverModal } from '../browser/approval-ux-modal';
+import { BrowserMessage } from '../browser/browser-in-use';
+import { PressableListItem, TitleListItem } from './list-items';
+
+export function DeveloperConsole() {
+  const { top, bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+  const contentOffsetTop = top + HEADER_HEIGHT;
+  const [getAddressesMessage, setGetAddressesMessage] = useState<BrowserMessage | null>(null);
+  return (
+    <Box
```

Comment:
Why not co-locate the routes and the pages? Seems odd to have these defined in two different places